### DISCUSSION TITLE
How to delete existing trillium installation?

### DISCUSSION THREAD

**Discussion by inikishev** [TriliumNext/Trilium#6030] [Ideas] [answered]
I downloaded the Zip file for "TriliumNextNotes-v0.90.8-windows-x64.zip". I unpacked it into my folder and launched "trilium.exe", after having launched "Squirrel.exe" because I wanted to see what it does. But I am greeted by my very old notes that I was taking a long time ago in the previous Trillium version, as well as old settings, etc. I would like to start over, but I am not quite sure how to do that. Where are all the settings located so that I can delete them?

**Comment by eliandoran** [comment, marked as answer]
@inikishev , if you start the TriliumNext application and press on the Trilium icon -> About Trilium, you can see a field for "Data directory". Clicking on it should open your file browser to the data directory. With the application closed simply remove or move everything and re-start the application and you should be good to go.
